source/translated_table_model.ts

```diff
@@ -115,12 +111,11 @@ export class TranslatedTableModel extends TableModel {
   }
 
   private moveDown(source: number, dest: number) {
-    const insides = this.translation.slice(source, dest + 1 );
-    const movedItem = insides.shift();
-    insides.push(movedItem);
-    this.translation =
-      this.translation.slice(
-        0,source).concat(insides.concat(this.translation.slice(dest + 1)));
+    const movingRow = this.translation[source];
```

Comment:
Merge ```moveDown``` and ```moveUp``` together into a single method. Refactor the code so that it works for both cases.